core/server/proxy/src/main/java/alluxio/proxy/s3/ListBucketResult.java

```diff
@@ -143,7 +144,7 @@ public ListBucketResult(
           S3ErrorCode.INVALID_ARGUMENT.getStatus()));
     }
     mDelimiter = options.getDelimiter();
-    if (mDelimiter != null) {
```

Comment:
Previously it can accept an empty string, now it can't. Had you tried emtpy string with/without this change?

Reply:
In fact, it only supports "/" as a delimiter.
<img width="1507" alt="image" src="https://user-images.githubusercontent.com/13930429/192250346-c25695eb-da55-458e-8d43-f4974ec8b9b6.png">